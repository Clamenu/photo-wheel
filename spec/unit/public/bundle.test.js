import React from 'react';
import PhotoWheel from '../../../public/bundle.js';
import { shallow } from 'enzyme';

describe('Photowheel component', () => {
  it('photos length is equal to 5', () => {
    const wrapper = shallow(<PhotoWheel />);
    const photos = wrapper.state().photos;
    expect(photos.length).toEqual(5);
  })
});