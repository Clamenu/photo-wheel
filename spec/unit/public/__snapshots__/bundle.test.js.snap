// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Photowheel component setup renders correctly 1`] = `
<div
  className="container"
  onClick={[Function]}
>
  <div
    className="test"
  >
    test
  </div>
  <br />
  <div
    className="container"
  />
</div>
`;
